### DISCUSSION TITLE
Programcı öğrenciler sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#207] [Zeka Sorusu] [answered]
Bir ortaokuldaki 45 öğrenci Java, C ve Python programlama dillerinden en az birini bilmektedir. Öğrencilerden 36 tanesi Java, 12 tanesi C ve 18 tanesi ise Python bilmektedir. 4 öğrenci bu üç dili de bildiğine göre bu dillerden en az iki tanesini bilen kaç öğrenci vardır?

**Comment by Berkek007** [comment, marked as answer]
Her üç dili de bilen 4 kişi olduğundan bunu toplamdan çıkartalım 45 - 4 = 41’dir. Java bilen 36 kişi ve her üç dili de bilen 4 kişi olduğu için 36 - 4 = 32’dir. C bilen 12 kişi ve her üç dili de bilen 4 kişi olduğundan 12 - 4 = 8’dir. Python bilen 18 kişi ve her üç dili de bilen 4 kişi olduğundan 18 - 4 = 14’tür. Bu üç cevap bizi 32 + 14 + 8 = 54 işlemine götürür ve bu öğrencilerin 54 dil bildiklerini (bazısı 2 ve bazısı 1'dir) anlayabiliriz. Bu cevaptan ilk bulduğumuz sayı olan 41'in (3'ten az dil bilenler) çıkartarak 54 - 41 = 13 elde ederiz, yani 2 dil bilenler
elde edilir. Bizden istenen bu dillerden en az iki tanesini bilen öğrenci sayısı olduğu için cevap 13 + 4 =17’dir.

cevap : 17
